seq2science/rules/merge_replicates.smk

```diff
@@ -83,9 +87,9 @@ if "replicate" in samples:
         input:
             unpack(get_merge_replicates),
         output:
-            temp(sorted(expand("{trimmed_dir}/merged/{{replicate}}{{fqext}}_trimmed.{fqsuffix}.gz", **config))),
+            temp(sorted(expand("{trimmed_dir}/{{replicate}}{{fqext}}_trimmed.{fqsuffix}.gz", **config))),
```

Comment:
This is a BIG change. With this, we no longer know which samples are original, and which are replicates. Furthermore, replicate names can no longer be the same as sample names.

Reply:
Nvm, only the first point stands, but I dont think thats important enough.